go.mod

```diff
@@ -0,0 +1,52 @@
+module github.com/kitex-contrib/config-apollo
+
+go 1.19
+
+require (
+	github.com/apolloconfig/agollo/v4 v4.3.1
+	github.com/cloudwego/kitex v0.7.2
+	github.com/cloudwego/kitex-examples v0.2.0
```

Comment:
Please make the subdirectory `example` an individual module by adding a `go.mod`, so we can remove the `kitex-examples` here.

Reply:
The example directory? How should I name the module, such as github.com/kitex-contrib/config-apollo/example?

Reply:
Or github.com/cloudwego/kitex-examples/kitex-contirb/config-apollo